### ISSUE TITLE
[BUG] rancher 2.13.1: this is not dark mode anymore

### ISSUE THREAD

**Issue by harridu** [rancher/dashboard#16264]
**Rancher Server Setup**
- Rancher version: 2.13.1
- Installation option (Docker install/Helm Chart): helm chart, RKE2

**User Information**
- What is the role of the user logged in? admin

**Describe the bug**
The new version provides new colors for the web GUI (at least in "dark mode"). The background switched from dark grey to grey, reducing contrast to the font color. Same goes for the buttons: They have a green background now. The chars in a slightly brighter green became less readable.

I think you missed the point of "dark mode". Colors became brighter, moving away from the black background and bright foreground colors that should have been provided. Contrast has been reduced. The new dark mode colors made the GUI more stressful for the eyes.

**Screenshots**

<img width="1350" height="706" alt="Image" src="https://github.com/user-attachments/assets/8509e17b-8ba5-42f8-b34c-2061f0d1ffda" />
<img width="1350" height="706" alt="Image" src="https://github.com/user-attachments/assets/6da9a108-3fe6-4035-a926-a328eda313d3" />

**Comment by snasovich** [comment]
@harridu , thank you for your report. Transferring to https://github.com/rancher/dashboard for UI team to take a look.

**Comment by rak-phillip** [comment]
@harridu thanks for raising this issue and helping to make Rancher better! We may reach out to you for additional information after we get this issue assigned for triage. 

In the mean time, you do have the ability to apply the original color scheme by setting the `ui-brand` setting to "classic" under Global Settings.

<img width="1563" height="295" alt="Image" src="https://github.com/user-attachments/assets/25eee4dd-16b9-41c2-8308-90be8c64d370" />

Does this help to alleviate some of the dark mode issues that you've encountered?

**Comment by nwmac** [comment]
Closing - we are continuing to refine the dark mode colors - please take a look at the latest Rancher version (2.15.0) and re-open if you still have suggestions for improvement.
